addons/point_of_sale/data/point_of_sale_onboarding.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<odoo>
+<odoo noupdate="1">
```

Comment:
seems unrelateed

Reply:
had an error due to this, trying to remove it.